File: yangxj96-serve/yangxj96-serve-gateway/src/main/java/io/github/yangxj96/server/gateway/Application.java
package io.github.yangxj96.server.gateway;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

/**
 * 前端网关的主启动类
 */
@MapperScan("io.github.yangxj96.server.gateway.mapper")
@EnableDiscoveryClient
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: yangxj96-serve-reactive/yangxj96-serve-demo/src/main/kotlin/io/github/yangxj96/server/reactive/demo/Application.kt
package io.github.yangxj96.server.reactive.demo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan

@EntityScan("io.github.yangxj96.server.reactive.demo.entity")
@ComponentScan(basePackages = [
    "io.github.yangxj96.server.reactive.demo.repos",
    "io.github.yangxj96.server.reactive.demo.controller",
    "io.github.yangxj96.server.reactive.demo.service"
])
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}